code/game/objects/items/clown_items.dm

```diff
@@ -110,10 +117,16 @@
 		skillcheck = user.mind.get_skill_modifier(/datum/skill/cleaning, SKILL_SPEED_MODIFIER)
 	if(prob(skillcheck*100)) //higher level = more uses assuming RNG is nice
 		uses--
-	if(uses <= 0)
+	if(uses <= 0 && !istype(src, /obj/item/soap/cyborg))
 		to_chat(user, span_warning("[src] crumbles into tiny bits!"))
 		qdel(src)
 
+/obj/item/soap/cyborg/decreaseUses(mob/user)
+	. = ..()
+	if(uses <= 0)
+		to_chat(user, span_warning("The soap has ran out of chemicals"))
```

Comment:
Rather then overriding `decreaseUses`, you should make a proc called like, `noUses`, something roughly around those lines.
You'd have it default to the current message and qdel, and here we could override it totally to just do what we want